### PR TITLE
Create fetch job

### PR DESCRIPTION
**Pull request by jcoyne**
Still needs:

- [x] retrieve WARCs
- [x] initiate accession WF

### CODE REVIEW

**Comment by ndushay** [comment]
@justinlittman do the empty checkboxes in the description mean anything, or have they been addressed?

File: config/settings/development.yml [hunk 1]
```diff
@@ -21,3 +21,5 @@ wasapi_providers:
         accountid: 1
         username: user
         password: pass
+crawl_directory: tmp/jobs
+wasapi_downloader_bin: /Users/jlittman/data/was/wasapi-downloader/build/install/wasapi-downloader/bin/wasapi-downloader
```
Comment (ndushay) [inline config/settings/development.yml]: I think you do NOT want your laptop dir here ...  maybe even "/add/local/dir/to/wasapi-downloader" or whatever.

Comment (justinlittman) [reply config/settings/development.yml]: Ooops. Forgot that.

**Comment by justinlittman** [comment]
#98 is for download / install of WASAPI downloader.

**Pull request merged**